docu.html

```diff
@@ -18,15 +18,15 @@
         <div class="videoDiv">
           <video class="backgroundVideo" autoplay loop muted>
             <source
-              src="./src/video/production ID 5193615.part"
+              src="src/video/production ID 5193615.mp4"
```

Comment:
Had problems with the video according to size since they werent compressed. videos have been uploaded to an external source but the links expire after a couple of days.
Will go for local solution for Capstone and solve external video story/compressing afterwards